translator/conventions/opencensus.go

```diff
@@ -12,18 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package tracetranslator
+package conventions
```

Comment:
translatorconventions?

Reply:
This is not a new package, it already exists, I just moved this file to that package.

Also I don't think Go recommends package names like that. I think using the base name is the recommendation, see https://golang.org/doc/effective_go.html#package-names

> By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps. Err on the side of brevity, since everyone using your package will be typing that name.

>Another convention is that the package name is the base name of its source directory;